models/Recording.js

```diff
@@ -763,6 +771,7 @@ module.exports = function(sequelize, DataTypes) {
   const apiUpdatableFields = ["location", "comment", "additionalMetadata"];
 
   Recording.processingStates = {
+    thermalTest: ["getMetadata", "toMp4", "FINISHED"],
```

Comment:
Am using it for a test case, as a way of getting exactly the same recording back after reprocessing. Probably isn't needed, but may pick up an error where the file processing get API isn't returning the reprocessed recording.

Reply:
ok... got it. 

I'm not a big fan of having test only features in the code. Another way to do this might be to have the test delete all recordings before it starts but that might be slow. 

At the very least please add a comment to explain why this is there.

Reply:
Yea it's not very nice being in there, I removed it and added a process_all_recordings function see comment. Seems to only take a second or 2 to run. So probably a better solution.